src/object/cnode.c

```diff
@@ -806,6 +806,21 @@ bool_t PURE isMDBParentOf(cte_t *cte_a, cte_t *cte_b)
         break;
     }
 
+#ifdef CONFIG_ALLOW_SMC_CALLS
+    case cap_smc_cap: {
+        word_t badge;
+
+        badge = cap_smc_cap_get_capSMCBadge(cte_a->cap);
+        if (badge == 0) {
+            return true;
+        }
+        return
```

Comment:
The formatting of this return is very weird. I know that cap_notification_cap does the same, but I don't think there is a reason to spread this further.

Reply:
Do you just mean that its weird that the keyword is on its own line?

Reply:
Yes.

Reply:
I thought that the original weirdness is due to an autostyle edge case. Turns out it's not, as the autostyle accepts it on the same line.. Agree we should change it.